spec/requests/api/graphql/graphql_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe API::GraphqlController, issues: ['railgun#147'] do
+
+  context 'Administrator can use users and user_roles' do
+    let(:authenticated_header) {
+      { 'Authorization' => "Bearer #{create(:administrator).api_token}" }
+    }
+
+    describe 'post /api/graphql#execute' do
+      it 'valid request returns users' do
```

Comment:
you make a lot of requests. instead, you can make one request with all fields, and check the answer. also use *let* feature of rspec to avoid code duplication